.github/actions/node/16.13.1/action.yml

```diff
@@ -0,0 +1,8 @@
+name: Node 16.13.1
+runs:
+  using: composite
+  steps:
+    - uses: actions/setup-node@v3
+      # channel.unsubscribe bug
+      with:
+        node-version: '16.13.1'
```

Comment:
It would be better to test with `v17.1.0`, the higher node version with bug? or one test with each major version with bug?